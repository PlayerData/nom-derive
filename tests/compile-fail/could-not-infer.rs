extern crate nom;
extern crate nom_derive;
use nom_derive::Nom;

#[derive(Nom)]
pub struct S {
    a: dyn Fn(u16) -> u16,
}

fn main() {}
