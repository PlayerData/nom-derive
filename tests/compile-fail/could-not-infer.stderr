error: Nom-derive: could not get first path ident
 --> $DIR/could-not-infer.rs:7:8
  |
7 |     a: dyn Fn(u16) -> u16,
  |        ^^^
